data/brands/shop/hearing_aids.json

```diff
@@ -180,6 +180,17 @@
         "shop": "hearing_aids"
       }
     },
+    {
+      "displayName": "Sonance Audition",
+      "id": "sonanceaudition-28a4ee",
+      "locationSet": {"include": ["fr"]},
+      "tags": {
+        "brand": "Sonance Audition",
+        "brand:wikidata": "Q114932560",
+        "network": "Sonance Audition",
```

Comment:
```suggestion
        "name": "Sonance Audition",
```

Reply:
as I said not all shops have the same namethat why I used network key